lib/plugins/aws/package/compile/events/s3/index.js

```diff
@@ -347,6 +350,7 @@ class AwsCompileS3Events {
                         Rules: rules,
                       },
                     ],
+                    ForceDeploy: forceDeployProperty,
```

Comment:
Let's add that property only if it's actually defined - that will allow us to avoid the unnecessary `ForceDeploy: undefined` in tests